examples/aggregator/module/rust/src/lib.rs

```diff
@@ -174,19 +182,59 @@ pub struct Config {
 
 oak::entrypoint!(oak_main => |in_channel| {
     oak::logger::init_default();
+
+    // Parse config.
     let config_map = oak::app_config_map(in_channel).expect("Couldn't read config map");
     let config: Config = toml::from_slice(&config_map.items.get("config").expect("Couldn't find config")).expect("Couldn't parse TOML config file");
     info!("Parsed config: {:?}", config);
-    let grpc_channel =
-        oak::grpc::server::init(&config.grpc_server_listen_address).expect("could not create gRPC server pseudo-Node");
-    let (invocation_sender, invocation_receiver) =
-        oak::io::channel_create::<oak::grpc::Invocation>()
-            .expect("could not create gRPC invocation channel");
-    oak::node_create(
-        &oak::node_config::wasm("app", "grpc_worker"),
-        invocation_receiver.handle,
-    ).expect("could not create gRPC worker node");
-    while let Ok(invocation) = grpc_channel.receive() {
+    let grpc_server_channel =
+        oak::grpc::server::init(&config.grpc_server_listen_address).expect("Couldn't create gRPC server pseudo-Node");
+
+    // Create an Aggregator Node.
+    let (init_sender, init_receiver) = oak::io::channel_create::<AggregatorInit>()
```

Comment:
I like the design in this PR. But compared to the multiplexing ideas using oneof discussed in #1246, is this design missing out anything?